### ISSUE TITLE
Create Customer View Account Flow 

### ISSUE THREAD

**Issue by bamboochow** [whynotearth/BrowTricks#153]
# Design File
For design reference, please see files in [Figma](https://www.figma.com/file/m8JgvnHpgPmGq8Bm2iAf1a/Brow-Tricks?node-id=709%3A10179)

See page "Tenant CMS (Prototype) 

# Customer View Account 
When a client is added to a salon owners contacts, salon owners are able to send them a unique link via email or text to complete their client profile. These pages are all existing pages in design, almost identical to the client/account pages of tenants, with limited access to certain information. This profile includes the following: 
- Log In Page 
- My Account 
- Settings 
- Client Notifications 
- Image Upload 
- PMU Question Flow 
- Account Activity 
- Payment Methods 
![image](https://user-images.githubusercontent.com/14948326/88762961-6eaa9d80-d19c-11ea-88c7-7e71502235e3.png)

# Log In Process 
Users are first sent an email/text (#94 PMU EMAIL CONFIRMATION, not part of this issue) with the link for them to access their account. Upon clicking the email link, they will be directed to their own login page. This login page is similar to the current log in page, with slightly different content 
![image](https://user-images.githubusercontent.com/14948326/88761953-58034700-d19a-11ea-955e-7b024e19bdbf.png)

### Content 
Title: Hello {client first name}! 
Text: In preparation for your appointment, please complete your client profile. 
Button: Log In With Google (google oAuth) 
![image](https://user-images.githubusercontent.com/14948326/88762195-e5df3200-d19a-11ea-98b2-b72a55724286.png)

# My Account & Settings 
After Log In, they are redirected to their account page. The account page is almost identical to the tenant's my acount page, with the exception that they do not have a business management section, and they have the additional expansions, which exist in client profile: 
- Notification Settings
- Images
- PMU creation flow

The settings page is the same design as the tenant settings page, with only these options: 
- my account 
- log out 

![image](https://user-images.githubusercontent.com/14948326/88762543-9cdbad80-d19b-11ea-876e-1e122cfd46de.png)

# Client Notification
This page is identical to existing client notification settings ` /tenant/:tenantSlug/clients/edit/:clientId/notifications `
https://github.com/whynotearth/BrowTricks/blob/master/src/views/ClientNotifications.vue
![image](https://user-images.githubusercontent.com/14948326/88763039-9e59a580-d19c-11ea-86f6-c26d16d2c330.png)

# Image Upload 
This page is identical to existing image upload page
` /tenant/:tenantSlug/clients/edit/:clientId/image-upload ` , with the exception of the "request images" button, which is tenant only. Uploaded images will be visible in a tenant's client profile gallery for that specific client. 
https://github.com/whynotearth/BrowTricks/blob/master/src/views/ClientImageUpload.vue

![image](https://user-images.githubusercontent.com/14948326/88763442-57b87b00-d19d-11ea-84b1-d2ddc37a0934.png)

# PMU Flow 
The PMU flow starts a little differently for customers. 
The PMU flow, if the client has not completed it, goes directly into the "Get Started page" `/tenant/:tenantSlug/client/:clientId/pmu/questions-intro` . The question flow itself is the same. https://github.com/whynotearth/BrowTricks/blob/master/src/views/PMUFlowQuestions.vue

![image](https://user-images.githubusercontent.com/14948326/88764003-5b98cd00-d19e-11ea-884c-cabe255dc113.png)

After completion, the customers are shown the success screen, and redirected to the PMU form preview page. The customer form preview page only shows AFTER completion of pmu form, and instead of "request completion" buttons, has a "download pdf" button for customers's own record. 

Once PMU form is complete, the "PMU" expansion in my accounts will show the preview instead of redirecting to the flow. Completed PMU form will be visible in the tenant's client profile as well. 
![image](https://user-images.githubusercontent.com/14948326/88764166-ad415780-d19e-11ea-8fdb-9b0d23eb6bd9.png)

# Activity & Transaction details 
Account activity/transaction details is the same design-wise as account activity for tenant's my account page (#84 Account Payment Activity #92 Transaction Detail ) . This will show all transactions with browtricks for that specific client. 
![image](https://user-images.githubusercontent.com/14948326/88764377-10cb8500-d19f-11ea-904c-d4b8abdd0fd0.png)

# Payment Methods
Customers are able to save payment methods that will be selectable in the tenant's payment feature in the future. They add payment methods the same way that tenants are currently designs to add payment methods. (#100 Tenant Payment Method) 
![image](https://user-images.githubusercontent.com/14948326/88764589-6869f080-d19f-11ea-897e-82657b8778ed.png)
